tests/sentry/api/test_event_search.py

```diff
@@ -1079,6 +1080,14 @@ def test_transaction_status_invalid(self):
         assert "Invalid value" in six.text_type(err)
         assert "cancelled," in six.text_type(err)
 
+    def test_trace_id(self):
+        result = get_filter("trace:{}".format("a0fa8803753e40fd8124b21eeb2986b5"))
+        assert result.conditions == [["trace", "=", uuid.UUID("a0fa8803753e40fd8124b21eeb2986b5")]]
+
+    def test_trace_span_id(self):
+        result = get_filter("trace.span:{}".format("8c931f4740435fb8"))
+        assert result.conditions == [["trace.span", "=", 10129474377767673784]]
```

Comment:
```suggestion
        assert result.conditions == [["trace.span", "=", int("8c931f4740435fb8", 16)]]
```

Reply:
It turns out that the span_id is already a string? At least, when I do this conversion it causes a type mismatch error in the tests.